clients/admin-ui/src/features/common/tablev2/cells.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  ArrowDownIcon,
+  ArrowUpIcon,
+  Box,
+  Checkbox,
+  Flex,
+  Text,
+} from "@fidesui/react";
+import { HeaderContext } from "@tanstack/react-table";
+import { HTMLProps, ReactNode, useState } from "react";
+
+export const DefaultCell = ({ value }: { value: string }) => (
+  <Flex alignItems="center" height="100%">
+    <Text fontSize="xs" lineHeight={4} fontWeight="normal">
+      {value}
+    </Text>
+  </Flex>
+);
+
+type IndeterminateCheckboxCellProps = {
```

Comment:
I didn't port over all of the cells. I think it makes more sense to gradually port them over as needed.


We don't need `FidesObject` anymore 🥳